lib/watir/wait.rb

```diff
@@ -71,20 +79,16 @@ def message_for(timeout, message)
       end
 
       def run_with_timer(timeout, &block)
-        if timeout == 0
```

Comment:
I added this logic to the `Timer#wait` method. The code for a given timer will always execute once, regardless of whether the timeout is set to `0` or `0.01`.